### ISSUE TITLE
Problem with the logfile produced by vina-gpu

### ISSUE THREAD

**Issue by xavgit** [DeltaGroupNJUPT/Vina-GPU-2.0#14]
Hi,
I have used vina-gpu with some ligands.

In the config file I have used the lines:
num_modes = 5
log = vina-gpu2_docking_res_small_sample_size.log

The first lines of the log file are the following:
\#################################################################
\# If you used Vina-GPU in your work, please cite:               #
\#                                                               #
\#Shidi, Tang, Chen Ruiqi, Lin Mengru, Lin Qingde,               #
\#Zhu Yanxiang, Wu Jiansheng, Hu Haifeng, and Ling Ming.         #
\#Accelerating AutoDock VINA with GPUs. ChemRxiv (2021).Print.   #
\#                                                               #
\#DOI 10.26434/chemrxiv-2021-3qvn2-v3                            #
\#                                                               #
\# And also the origin AutoDock Vina paper:                      #
\# O. Trott, A. J. Olson,                                        #
\# AutoDock Vina: improving the speed and accuracy of docking    #
\# with a new scoring function, efficient optimization and       #
\# multithreading, Journal of Computational Chemistry 31 (2010)  #
\# 455-461                                                       #
\#                                                               #
\# DOI 10.1002/jcc.21334                                         #
\#                                                               #
\#################################################################

Reading input ... done.
Setting up the scoring function ... done.
Analyzing the binding site ... done.
done.
mode |   affinity | dist from best mode
     | (kcal/mol) | rmsd l.b.| rmsd u.b.
-----+------------+----------+----------
   1         -6.1      0.000      0.000
   2         -6.1      3.182      7.879
   3         -6.1      2.692      5.399
   4         -6.1      2.460      5.991
   5         -6.0      2.067      4.375
done.
done.
mode |   affinity | dist from best mode
     | (kcal/mol) | rmsd l.b.| rmsd u.b.
-----+------------+----------+----------
   1         -6.5      0.000      0.000
   2         -6.4      2.546      4.985
   3         -6.3      2.380      6.271
   4         -6.3      2.525      6.562
   5         -6.3      2.457      4.470
done.
done.
mode |   affinity | dist from best mode
     | (kcal/mol) | rmsd l.b.| rmsd u.b.
-----+------------+----------+----------
   1         -7.2      0.000      0.000
   2         -7.2      2.848      6.771
   3         -6.8      2.888      6.742
   4         -6.8      2.391      6.465
   5         -6.7      3.107      6.380
done.
done.
mode |   affinity | dist from best mode
     | (kcal/mol) | rmsd l.b.| rmsd u.b.
-----+------------+----------+----------
   1         -6.4      0.000      0.000
   2         -6.2      3.115      6.358
   3         -6.1      3.824      5.667
   4         -6.1      3.818      5.903
   5         -6.1      3.785      7.754
done.
done.

The problem is that the log file lacks the information relative to which ligand the section

mode 
.......
.......
done.
done.

 is referred to.
In other words I receive docking results but I don't know which ligand has which result.

If I'm not wrong can I receive some help to solve this problem?

Thanks.
Saverio

**Comment by wang-sy** [comment]
I have the same question
